lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
+      @_installDependencies
+      @_writeMychaConfig
+      @_writeTestHelper
+    ], done
+
+
+  _installDependencies: (done) =>
+    exec "npm install --save-dev chai sinon sinon-chai", {@cwd}, done
+
+
+  _writeMychaConfig: (done) =>
```

Comment:
mycha currently isn't meant to be installed globally. It still has mocha as a dependency so you need it installed locally to run tests on ci.